### PR TITLE
Add new design system links

### LINKED ISSUE #20860: Reference in browser extension documentation to obsolete "Iconography" page
### MDN URL

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_action

### What specific section or headline is this issue about?

Icons

### What information was incorrect, unhelpful, or incomplete?

There's a link to an Iconography page ([destination](https://design.firefox.com/photon/visuals/iconography.html)) which declares itself to be no longer maintained since the 2021 design refresh release.

### What did you expect to see?

A link to a current and maintained page.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/mozilla/add-ons/webextensions/user_interface/browser_action`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_action
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/add-ons/webextensions/user_interface/browser_action/index.md
* Last commit: https://github.com/mdn/content/commit/1ee49b729dc4cd02b862d910f279861f4b30e704
* Document last modified: 2022-09-09T05:03:45.000Z

</details>

### PR DESCRIPTION
**Pull request by dawei-wang**
Fixed two links that pointed to the same page. Updates the links to point to relevant pages in the Acorn design system. 
Fix mdn/content#20860

### CODE REVIEW

**Comment by rebloor** [comment]
@dawei-wang would you consider fixing the other out-of-date links in:

- files/en-us/mozilla/add-ons/webextensions/manifest.json/options_ui/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/context_menu_items/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/devtools_panels/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/extension_pages/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/notifications/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/options_pages/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/page_actions/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/popups/index.md
- files/en-us/mozilla/add-ons/webextensions/user_interface/sidebars/index.md

**Comment by dawei-wang** [comment]
> @dawei-wang would you consider fixing the other out-of-date links in:
> 
> * files/en-us/mozilla/add-ons/webextensions/manifest.json/options_ui/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/context_menu_items/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/devtools_panels/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/extension_pages/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/notifications/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/options_pages/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/page_actions/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/popups/index.md
> * files/en-us/mozilla/add-ons/webextensions/user_interface/sidebars/index.md

Sure!

File: files/en-us/mozilla/add-ons/webextensions/user_interface/options_pages/index.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ See the [`options_ui`](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/o
 
 ## Options content design
 
-For details on how to design your options content to match the style of Firefox, see the [Photon Design System](https://design.firefox.com/photon/index.html) and [browser styles](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles) documentation.
+For details on how to design your options content to match the style of Firefox, see the [Acorn Design System](https://acorn.firefox.com/latest/acorn.htmll) and [browser styles](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles) documentation.
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/user_interface/options_pages/index.md:61]:
```suggestion
For details on how to design your options content to match the style of Firefox, see the [Acorn Design System](https://acorn.firefox.com/latest/acorn.html) and [browser styles](/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles) documentation.
```

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
@dawei-wang Thanks for fixing them all,

**Pull request merged**